src/test/EigenPod.t.sol

```diff
@@ -265,6 +265,13 @@ contract EigenPodTests is ProofParsing, EigenPodPausingConstants {
         cheats.stopPrank();
     }
 
+    function testStakingWithInvalidAmount () public {
```

Comment:
can you move this test to EigenPodUnit.t.sol and then do the test with pod.stake instead of podManager.stake